libfuse/constants_darwin.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
```

Comment:
may as well make this +build darwin, so we can catch errors where we try to use a darwin-only constant in a non-darwin file